### ISSUE TITLE
Form groups with insufficent group labels [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#259]
The “Select your country”, “Gender identity”, “Age range”, “Race”, “Ethnicity” overlays accessed from the My Health Log About screen all have the same insufficient group label of “Choose…”, which doesn’t sufficiently describe the purpose of the form group.

![“Choose…” form group label focused with VoiceOver](https://user-images.githubusercontent.com/4437084/92042798-ee51fc00-ed48-11ea-96b5-26ae333a5b0b.png)

The “Select your country” overlay accessed from the App Settings -> My Health Log screen has the insufficient group label of “placeholder”.

![“Placeholder” form group label focused with VoiceOver](https://user-images.githubusercontent.com/4437084/92042817-f742cd80-ed48-11ea-850f-eacd31462434.png)

Ensure all form group labels sufficiently describe the purpose of the form group (e.g., “Choose your gender identity”).

**Comment by willslone** [comment]
I retested and can confirm the issue has been resolved.
